src/Orleans/Messaging/ByteArrayBuilder.cs

```diff
@@ -440,23 +447,6 @@ public ByteArrayBuilder Append(double i)
             return this;
         }
 
-        public ByteArrayBuilder Append(Guid g)
```

Comment:
This code was not used. There is another code, in BinaryTokenStreamWriter doing the same.